src/target/riscv/riscv.c

```diff
@@ -1481,8 +1494,10 @@ int riscv_flush_registers(struct target *target)
 	return ERROR_OK;
```

Comment:
For easier orientation in the debug log (to be able to easily find when the cache flush started and when it finished), you can add a debug print also to the end of the function:

```suggestion
	LOG_TARGET_DEBUG(target, "Flush of register cache completed");
	return ERROR_OK;
```

Reply:
Addressed.